src/main/java/edu/ucsb/cs156/example/controllers/HotelController,java

```diff
@@ -0,0 +1,105 @@
+package edu.ucsb.cs156.example.controllers;
+
+import edu.ucsb.cs156.example.entities.Hotels;
+import edu.ucsb.cs156.example.errors.EntityNotFoundException;
+import edu.ucsb.cs156.example.repositories.HotelsRepository;
```

Comment:
same issue here, your repository file is "HotelRepository" but you used "HotelsRepository" here, change throughout file as well